agent/pkg/tunnel/tunnel.go

```diff
@@ -13,21 +13,23 @@ import (
 	"github.com/kubeedge/edgemesh/common/constants"
 )
 
-const RETRY_CONNECT_TIME = 3
+const connectRetryTime = 3
+const runRetryTime = 10
 
 func (t *TunnelAgent) Run() {
-	for {
+	isSuccess := false
+	for i := 0; i < runRetryTime; i++ {
 		relay, err := controller.APIConn.GetPeerAddrInfo(constants.ServerAddrName)
```

Comment:
need to reset `i` when calling GetPeerAddrInfo successfully?

Reply:
I do not think it is necessary. And if the code reset i when calling GetPeerAddrInfo successfully , this may cause the infinite loop

Reply:
A question: runRetryTime means retry what?

Reply:
retry the whole Run function

Reply:
`Run` is an action, what does it really mean? retry to GetPeerAddrInfo until it succeeed, OR connect server?